_components/identifier/guidance/when-to-use.md

```diff
@@ -1,2 +1,2 @@
-- **To identify a site's parent agency.** The identifier is a complement to the USWDS [banner]({{ site.baseurl }}/components/banner). Use the identifier to tell users what agency is responsible for your website.
+- **To identify highest-level agency associated with a site or service.** The identifier is a complement to the USWDS [banner]({{ site.baseurl }}/components/banner). Use the identifier to tell users what agency is responsible for your website. If your site is the primary site associated with an agency, you can still use the identifier. Consider the parent agency to be the highest-level agency associated with a site or service.
```

Comment:
Recommend revising so that this section reads as follows:

**To identify the highest-level agency associated with a site or service.** The identifier is a complement to the USWDS [banner](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/dw-identifier-parent/components/banner). Use the identifier to tell users what parent agency is responsible for your website. Consider the parent agency the highest-level agency associated with a site or service. If your site is the primary site associated with an agency, you can still use the identifier.

Here's what I changed:

- Add “the” after “identify” in first sentence/intro.
- Add “parent” before “agency” in the third sentence.
- Switch the order of the last two sentences. 
- Remove “to be” from what is now the penultimate sentence ("Consider the parent agency the highest…")

Reply:
```suggestion
- **To identify the highest-level agency associated with a site or service.** The identifier is a complement to the USWDS [banner]({{ site.baseurl }}/components/banner). Use the identifier to tell users what parent agency is responsible for your website. Consider the parent agency the highest-level agency associated with a site or service. If your site is the primary site associated with an agency, you can still use the identifier. 
```

Reply:
Adding edit suggestions, but leaving `can` as-is